ui/lib/core/addon/templates/components/layout-loading.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="is-flex-v-centered is-flex-1 loader-inner-page" >
+<div class="is-flex-v-centered is-flex-1 loader-inner-page" data-test-layout-loading>
```

Comment:
this def doesn't need to change, but as an FYI since this component already has a unique class name like `loader-inner-page`, it doesn't necessarily need a `data-test-*` attribute. comes down to personal preference, i suppose :D

Reply:
Ah good to know.  Any chance you can add this to your Ember learn on testing 🙂📖